clients/go/apis/v1alpha1/config_test.go

```diff
@@ -375,20 +375,25 @@ func TestSetDefault(t *testing.T) {
 		wantCfg *Config
 	}{{
 		name: "default_version",
-		cfg:  &Config{},
```

Comment:
Can you add some tests for different casing of the log mode? I think if the user puts "best_effort" instead of "BEST_EFFORT", it will fail right now (L140)

Reply:
Done